config/detekt/detekt.yml

```diff
@@ -0,0 +1,808 @@
+build:
```

Comment:
Do we need to turn off/on some rules here?

Reply:
My suggestion is to use:

https://github.com/game-x50/android_client_app/blob/master/build.gradle.kts#L57

https://github.com/game-x50/android_client_app/blob/master/gradle_support/linters.gradle

https://github.com/game-x50/android_client_app/blob/master/linters/detekt-config.yml

Reply:
Done, added.